torchbenchmark/util/framework/huggingface/patch_hf.py

```diff
@@ -5,16 +5,22 @@
 import subprocess
 import sys
 from .model_factory import class_models
-from transformers import AutoConfig, ReformerConfig, BigBirdConfig, BertConfig, WhisperConfig, LlamaConfig
-
+from transformers import AutoConfig, ReformerConfig, BigBirdConfig, BertConfig, WhisperConfig, LlamaConfig, PhiConfig
+import inspect
 
 PATCH_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), "patches")
 
 def cache_model(name: str, **kwargs):
     import transformers
     model_config = eval(class_models[name][2])
     model_ctor = getattr(transformers, class_models[name][3])
-    model_ctor.from_config(model_config, **kwargs)
+    print(model_config.__class__.__name__)
+    print("~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~")
+    if (model_config.__class__.__name__ is "PhiConfig" or "LlavaConfig"):
```

Comment:
I am wondering why we can't use the same code from `model_factory.py` to determine whether `from_config` should be used?

Reply:
Causes errors w/ runs for other HF models (feel free to test & replicate)